src/commercial.consentless.ts

```diff
@@ -20,15 +23,26 @@ const bootConsentless = async (consentState: ConsentState): Promise<void> => {
  	*/
 	maybeUnsetAdFreeCookie(AdFreeCookieReasons.ConsentOptOut);
 
-	await Promise.all([
+	const consentlessModuleList = [
 		setAdTestCookie(),
 		setAdTestInLabelsCookie(),
 		initConsentless(consentState),
 		initExclusionSlot(),
 		initFixedSlots(),
 		initArticleInline(),
 		initLiveblogInline(),
-	]);
+	];
+
+	if (isDotcomRendering) {
+		const userFeatures = await import(
+			/* webpackChunkName: "dcr" */
+			'lib/user-features'
+		);
+
+		consentlessModuleList.push(userFeatures.refresh());
```

Comment:
Do we want to push the function here rather than invoking?

```suggestion
		consentlessModuleList.push(userFeatures.refresh);
```

Reply:
Ah okay sorry yes its okay as you have it. We have two different methods of booting in standalone and consentless. In `commercial.standalone` we create an array of functions which we later invoke but in `commercial.consentless` we immediately execute a bunch of promises that we `promise.all` on.